bottom-rs.mk

```diff
@@ -0,0 +1,39 @@
+ifneq ($(PROCURSUS),1)
+$(error Use the main Makefile)
+endif
+
+SUBPROJECTS       += bottom-rs
+BOTTOM-RS_COMMIT  := 3451cdadd7c4e64fe8e7f43e986a18628a741dec
+BOTTOM-RS_VERSION := 1.0+git20210404.$(shell echo $(BOTTOM-RS_COMMIT) | cut -c -7)
```

Comment:
Change this to 1.2.0 (without the git hash and all that) to match the version number of the actual bin. This might change If I manage to convince Kaylynn to finally start using tags...